ci/all-docker.sh

```diff
@@ -31,7 +31,7 @@ if [[ -z "${STACK_NAME}" ]]; then
 fi
 AZ="${AZ:-us-west-2c}"
 REGION="${REGION:-us-west-2}"
-S3_BUCKET="${S3_BUCKET:-"heptio-ci-aws-quickstart"}"
+S3_BUCKET="${S3_BUCKET:-"vmware-ci-aws-quickstart"}"
```

Comment:
This is the line that breaks tests currently but I figure we do want to change it and fix s3, correct?